library/src/main/res/values/dimens.xml

```diff
@@ -9,4 +9,7 @@
     <dimen name="info_window_nonfocus_text_size">14dp</dimen>
     <dimen name="info_window_focus_padding">14dp</dimen>
     <dimen name="info_window_nonfocus_padding">10dp</dimen>
+    <dimen name="my_location_button_dimen">48dp</dimen>
+    <dimen name="hamburger_margin">24dp</dimen>
```

Comment:
This should not be in the **library** but in the **sample** module

Reply:
Removed.